File: 598_range-addition-ii.cpp
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>>& ops) {
        int row = m, col = n;
        for (auto& op : ops) {
            row = min(row, op[0]);
            col = min(col, op[1]);
        }
        return row * col;
    }
};

File: 598_range-addition-ii.py
class Solution(object):
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        return m * n if not ops else min([op[0] for op in ops]) * min([op[1] for op in ops])
